docs/api/RESTAPI.md

```diff
@@ -779,9 +783,13 @@ Workflow query parameters
 |Name|Description|Schema|
 |---|---|---|
 |**end**  <br>*optional*|Returns only workflows with an equal or earlier end datetime.  Can be specified at most once. If both start and end date are specified, start date must be before or equal to end date.|string (date-time)|
+|**excludeLabelAnd**  <br>*optional*|Excludes workflows with the specified label.  If specified multiple times, excludes workflows with all of the specified label keys. Specify the label key and label value pair as separated with "label-key:label-value"  <br>**Pattern** : `"^([a-z][-a-z0-9]*[a-z0-9])?[:]([a-z][-a-z0-9]*[a-z0-9])?$"`|string (array)|
+|**excludeLabelOr**  <br>*optional*|Excludes workflows with the specified label.  If specified multiple times, excludes workflows with any of the specified label keys. Specify the label key and label value pair as separated with "label-key:label-value"  <br>**Pattern** : `"^([a-z][-a-z0-9]*[a-z0-9])?[:]([a-z][-a-z0-9]*[a-z0-9])?$"`|string (array)|
 |**id**  <br>*optional*|Returns only workflows with the specified workflow id.  If specified multiple times, returns workflows with any of the specified workflow ids.  <br>**Pattern** : `"^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$"`|string|
 |**includeSubworkflows**  <br>*optional*|Include subworkflows in results. By default, it is taken as true.|string (boolean)|
 |**name**  <br>*optional*|Returns only workflows with the specified name.  If specified multiple times, returns workflows with any of the specified names.  <br>**Pattern** : `"^[a-zA-Z][a-zA-Z0-9_]*$"`|string|
+|**page**  <br>*optional*|Page number for results (for paginated query)|string (integer)|
+|**pagesize**  <br>*optional*|Number of results per page (for paginated query)|string (integer)|
```

Comment:
why are these being added to the doc and swagger now?

Reply:
I am not sure how long these 2 parameters have been included in query endpoint, but I was just trying to update the Swagger doc (for query endpoint). 

Since I always use Swagger to know data models of request/response and make API requests, it was a bit of surprise for me when I found out last week that there exists page and pagesize parameters in query and I never saw them Swagger UI.

I can remove them if it seems unnecessary to add it here.

Reply:
I think "because they weren't here before" is a totally valid answer!

As long as the fields work when used in swagger, I'd 100% keep them (unless we were trying to hide their existence for some reason?)

Reply:
The fields do work when used in Swagger. I did test it. Though, I am not sure why it was not included before!